gilbreth_support/launch/gilbreth.launch

```diff
@@ -27,7 +27,9 @@
   <include file="$(find gilbreth_support)/launch/spawn.launch"/>
```

Comment:
If this is the launch file that runs the whole show maybe it should be renamed to "run_application.launch" or something along those lines.